packages/integration-sdk-runtime/src/synchronization/index.ts

```diff
@@ -180,6 +180,8 @@ export async function initiateSynchronization(
     job,
     logger: logger.child({
       ...jobConfiguration,
+      integrationJobId:
+        jobConfiguration.integrationJobId ?? job.integrationJobId,
```

Comment:
Will `job.integrationJobId` ever be unset? Is it enough to simply do:

```suggestion
      integrationJobId: job.integrationJobId,
```

Reply:
I'm not sure. In theory if jobConfiguration.integrationJobId is set, then the syncJob's job.integrationJobId _should_ be equal to whatever jobConfiguration.integrationJobId  is. But just in case, I went with this since it will ensure the functionality is the same as it was before, just with the included fallback.